pkg/client/eventsbuilder.go

```diff
@@ -31,6 +33,12 @@ func (builder *eventsBuilder) getStartPosition() *orderer.SeekPosition {
 
 type eventOption = func(builder *eventsBuilder) error
 
+// Inferace used while checkpointing events
```

Comment:
We can fix this later when we check all the docs, but godoc comments must start with the name of the thing they are commenting. So The first word of this comment must be **Checkpointer**. See https://go.dev/blog/godoc